src/Canvanoid.js

```diff
@@ -17,39 +17,93 @@ export default class Canvanoid {
 					  delta: null }
 
 	    this.messagePanel = null;
-		this.score = null;
+		this.instructionsPanel = null;
+		this.scorePanel = null;
 
 		this.state = null; 
 		this.board = null;
 		this.balls = null;
 		this.vaus = null; 
+		this.pause = null;
 	}
 	
 	start() {	
+		// Game Elements
 		this.state = new State();
-		
 		this.board = new Board();
 		this.board.setStage(this.state.stage);
 
-		this.balls = [ new Ball(this.board.x + this.board.w/2, this.board.y + this.board.h/2 + 100) ];
-		this.vaus = new Vaus(this.board.x + this.board.w/2 - 50, 
-              				 this.board.y + this.board.h - 50);
-		this.vaus.start();
+		this.balls = [ new Ball(this.board.position.x + this.board.width/2, this.board.position.y + this.board.height/2 + 100) ];
+		this.vaus = new Paddle(this.board.position.x + this.board.width/2 - 50, 
+              				   this.board.position.y + this.board.height - 50);
+		this.interruptions();
 
-		this.score = new Score(this.board.x + this.board.w - 100, 
-		                       this.board.y + this.board.h + 30);
+		// Interface Elements
+		this.messagePanel = new Panel(this.board.position.x + this.board.width/2, 
+							          this.board.position.y + this.board.height/2,
+							   		  this.ctx);
 		
-		this.messagePanel = new Panel( this.board.x + this.board.w/2 - 150, 
-							           this.board.y + this.board.h/2 - 100);
+		this.instructionsPanel = new Panel(this.messagePanel.initialPosition.x, this.messagePanel.initialPosition.y + 50,
+							         	   this.ctx);
+		this.instructionsPanel.setSize("18");
```

Comment:
I'm not sure of how not to hardcode these consts.

I could make global variables (e.g.: _small_font_size, _big_font_size...) and use them to replace the "18" and other similar consts. However, they would still be passed to the function the same way -although I get why it makes the code more comprehensible. Would that be okay? Were you thinking of something else?

On this topic, the panels are often aligned left, center or right when needed. These are strings passed into the function, which causes the panel to rearrange itself. Would we consider them ("left", "center") hardcoded consts?